Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -111,6 +111,12 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
         }
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        coordinator.animate(alongsideTransition: { context in
+            self.presentedViewController?.dismiss(animated: true, completion: nil)
```

Comment:
Yea. It actually doesnt look bad if you animate it. If you do the animations inside that coordinator block it'll merge the animations together and look fine.